README.md

```diff
@@ -1 +1,98 @@
-# Things3 Low-Level Python API to the Database
\ No newline at end of file
+Get your Things app data into a Python dict.
+
+## Install
+```python
+$ pip install things.py
+```
+
+## Run
+```pycon
+>>> import things
+>>> things.tasks()
+[{'uuid': '2Ukg8I2nLukhyEM7wYiBeb',
+  'type': 'task',
+  'title': 'Make reservation for dinner',
+  'project': {'title': 'Throw Birthday Party', 'uuid': 'bNj6TPdKYhY6fScvXWVRDX'},
+  ...},
```

Comment:
status attribute here with a number or a string?

Reply:
Yes! "status" is in {"open", "cancelled", "done"}